### ISSUE TITLE
Apple silicon local docker gochain issue 

### ISSUE THREAD

**Issue by redlarva** [icon-project/goloop#93]
I followed the instruction given on [gochain-local](https://github.com/icon-project/gochain-local), however unable to run docker gochain-iconee container. It exits with the following error message

```
I|20220318-11:34:15.977207|b6b5|-|pyee|pyee.go:172 start instance uid=580562a8-c8c5-4056-bb9a-1304fe87f324
I|20220318-11:34:15.995392|b6b5|-|chain|chain.go:421 ConfigFilepath /testsuite/data/config/icon.json BaseDir ../../chain/iconee ChainDir /testsuite/chain/iconee
I|20220318-11:34:15.995744|b6b5|3325fd|chain|chain.go:357 prepare a directory /testsuite/chain/iconee/db for database
P|20220318-11:34:16.003668|b6b5|-|main|main.go:480 FAIL to initialize Chain err=fail to open database dir=/testsuite/chain/iconee/db type=rocksdb name=3
github.com/icon-project/goloop/common/errors.Wrapf
	/work/common/errors/errors.go:232
github.com/icon-project/goloop/chain.(*singleChain).openDatabase
	/work/chain/chain.go:364
github.com/icon-project/goloop/chain.(*singleChain).prepareDatabase
	/work/chain/chain.go:382
github.com/icon-project/goloop/chain.(*singleChain)._init
	/work/chain/chain.go:429
github.com/icon-project/goloop/chain.(*singleChain).Init
	/work/chain/chain.go:527
main.Execute
```

 **System Software Overview:**
```
      System Version: macOS 12.0 (21A344)
      Kernel Version: Darwin 21.0.1
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: MacBook Pro
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
```

**Hardware Overview:**
```
      Model Name: MacBook Pro
      Model Identifier: MacBookPro18,3
      Chip: Apple M1 Pro
      Total Number of Cores: 8 (6 performance and 2 efficiency)
      Memory: 16 GB
      System Firmware Version: 7429.30.65
      OS Loader Version: 7429.30.65

```

[iconee.log](https://github.com/icon-project/goloop/files/8308872/iconee.log)

**Comment by jspark-icon** [comment]
Currently, build-script of goloop is fit of amd64 build, thus it's not support arm64 build without any modification.
To amd64 build on Apple silicon,  set as `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before `make gochain-icon-image`
Before rebuild, you should remove all output of arm64 build as below.
* Remove all related docker images that 'goloop/' prefixed
* Remove build-cache of docker by `docker builder prune -a`
* Remove `goloop/build` directory
* Remove build output of javaee by `cd goloop/javaee; ./gradlew clean`

**Comment by sink772** [comment]
Fixed in 30b3f02f4b32b3726e502edce6d9c22a2f2afb28
